### ISSUE TITLE
Engine version 1.1.14202.0 fails calling DllMain() 

### ISSUE THREAD

**Issue by earada** [taviso/loadlibrary#36]
After updating the engine to version 1.1.14202.0, mpclient isn't able to load DllMain.

```
root@093c610a27cc:/loadlibrary$ gdb -q ./mpclient
Reading symbols from ./mpclient...done.
(gdb) r /eicar.txt
Starting program: /loadlibrary/mpclient /eicar.txt
MultiByteToWideChar(): Unsupported Conversion Flags 0x1
MultiByteToWideChar(): Unsupported Conversion Flags 0x1
MultiByteToWideChar(): Unsupported Conversion Flags 0x1
MultiByteToWideChar(): Unsupported Conversion Flags 0x1
MultiByteToWideChar(): Unsupported Conversion Flags 0x1
MultiByteToWideChar(): Unsupported Conversion Flags 0x1

Program received signal SIGSEGV, Segmentation fault.
0xf6d35fbe in ?? ()
(gdb) bt
#0  0xf6d35fbe in ?? ()
#1  0xf6d360f3 in ?? ()
#2  0xf6d34e05 in ?? ()
#3  0xf6d32bf0 in ?? ()
#4  0xf6d32bc5 in ?? ()
#5  0xf6d32dff in ?? ()
#6  0xf6d32f06 in ?? ()
#7  0x56557013 in main (argc=<optimized out>, argv=<optimized out>, envp=<optimized out>) at mpclient.c:193
```

**Comment by taviso** [comment]
Thanks for the excellent bug report, I think there are two bugs:

- I hadn't labeled some of the locale functions with the correct calling convention (and neither did MSDN...grrr).
- The latest build uses the function `GetProcessTimes`, so I needed to add that.

I think I made `MultiByteToWideChar` handle that conversion flag as well, although it would still have worked okay with some minor string mangling.
